src/sql/sqlClient.js

```diff
@@ -1,11 +1,22 @@
 const knex = require('knex');
+const { types } = require('pg');
+
+const TIMESTAMPZ = 1184;
+
+const setCustomParsers = () => {
+  // Default timestampz parser returns an object which breaks swagger's validation
+  // With this we can override that parser to return a string
+  types.setTypeParser(TIMESTAMPZ, (dateString) => dateString);
```

Comment:
Does this also set the returned object's type to string?

Reply:
dateString is a string already